demo/src/components/CTodos.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+    <div>
+        <h1>ToDo List</h1>
+        <p>This basic example shows you how the Undo/Redo Vuex plugin works. Simply add your ToDo to the input field and
+            hit the "Enter" key. You can then undo and redo those actions.</p>
+        <form action="#">
+            <input v-model="newTodo" type="text" placeholder="Add todo" @keyup.enter.prevent="addTodo">
```

Comment:
[user-testing] This does not correctly prevent form submission the first time if executed from localhost:8080